### PR TITLE
RHDEVDOCS-6131: Replace instances of RolloutManager with Rollout

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps 1.13, GitOps 1.14

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6131

Link to docs preview:

Step 3.d of the section [Configuring Argo Rollouts to route traffic by using OpenShift Routes](https://79915--ocpdocs-pr.netlify.app/openshift-gitops/latest/argo_rollouts/routing-traffic-by-using-argo-rollouts.html#gitops-configure-rollout-route-traffic-using-openshift-routes_routing-traffic-by-using-argo-rollouts)

QE review:
- [ ] QE has approved this change.

SME review: @jgwest 
QE review: @varshab1210 
Peer review: @GroceryBoyJr 

Additional information:

### CODE REVIEW

**Review by jgwest — APPROVED** [review]
LGTM, thanks @Dhruv-Soni11!

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM. One minor suggestion!

File: modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc [hunk 1]
```diff
@@ -173,9 +173,7 @@ spec:
 <5> Specify the steps for the rollout. This example gradually routes 30%, 60%, and 100% of traffic to the canary version.
 <6> Ensure that the contents of the `selector` field are the same as in canary and stable service.
 .. Click *Create*.
-+
-After the rollout has been created, you can verify that the *Status* field of the rollout shows *Phase: Healthy*.
-.. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the *RolloutManager* instance shows as *Phase: Available*.
+.. In the *Rollout* tab, under the *Rollout* section, verify that the *Status* field of the rollout shows as *Phase: Healthy*.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc:176]: Fixed.

**Review by skrthomas — COMMENTED** [review]
merge review lgtm

**Pull request merged**